yarn.lock

```diff
@@ -5733,9 +5733,10 @@ typedarray@^0.0.6:
   version "0.0.6"
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
 
-typescript@3.2.2:
```

Comment:
This is for later - but we should probably establish some way of dealing with lockfile updates... maybe add a bot to the repo to deal with dependencies and remove the lockfile from git tracking?